### PR TITLE
Fixes and improvements/fixes/#11

### LINKED ISSUE #23: design improvements

### LINKED ISSUE #25: bugfixes for games page

### PR DESCRIPTION
**Pull request by drmodun**
closes #23, closes #25
Mostly bugfixes and design improvements

### CODE REVIEW

File: pub-critic/src/app/components/Filter/Filter.tsx [hunk 1]
```diff
@@ -87,6 +107,9 @@ export const Filter = ({
             onSelect={(value) => {
               setGenre(value as number);
             }}
+            initSelected={
+              genres.find((selectedGenre) => selectedGenre.id == genre)?.name
```
Comment (ilovric00) [inline pub-critic/src/app/components/Filter/Filter.tsx]: Maybe is better for performance to move these calculations outside of rendering part. Also will be easier to read.

File: pub-critic/src/app/components/HomepageSection/HomepageSection.module.scss [hunk 2]
```diff
@@ -47,10 +46,13 @@
     align-items: flex-start;
     justify-content: flex-start;
     overflow-x: scroll;
-    margin-top: 20px;
+
     > div {
-      margin-right: vw-calc(20);
+      position: relative;
+      top: vw-calc(-40px);
```
Comment (ilovric00) [inline pub-critic/src/app/components/HomepageSection/HomepageSection.module.scss]: remove `px` part

File: pub-critic/src/app/components/LoadingSpinner/Spinner.module.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@keyframes spinner {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+.spinner {
```
Comment (ilovric00) [inline pub-critic/src/app/components/LoadingSpinner/Spinner.module.scss:10]: add some space :)

File: pub-critic/src/app/games/page.module.scss [hunk 4]
```diff
@@ -6,9 +6,10 @@
   min-height: 100vh;
   display: flex;
   flex-direction: column;
+  align-items: flex-start;
   justify-content: center;
   background-color: $background-color;
-  padding: vw-calc(70) vw-calc(20);
+  padding: vw-calc(70) vw-calc(10);
```
Comment (ilovric00) [inline pub-critic/src/app/games/page.module.scss]: `padding: vw-calc(70 10);`

**Review by ilovric00 — APPROVED** [review]

File: pub-critic/src/app/components/HomepageSection/HomepageSection.module.scss (hunk 2 above)
Comment (drmodun) [reply pub-critic/src/app/components/HomepageSection/HomepageSection.module.scss]: :white_check_mark:

File: pub-critic/src/app/components/LoadingSpinner/Spinner.module.scss (hunk 3 above)
Comment (drmodun) [reply pub-critic/src/app/components/LoadingSpinner/Spinner.module.scss:10]: :white_check_mark:

File: pub-critic/src/app/games/page.module.scss (hunk 4 above)
Comment (drmodun) [reply pub-critic/src/app/games/page.module.scss]: :white_check_mark:

File: pub-critic/src/app/components/Filter/Filter.tsx (hunk 1 above)
Comment (drmodun) [reply pub-critic/src/app/components/Filter/Filter.tsx]: :white_check_mark:

**Comment by drmodun** [comment]
Since the demo is tommorow, I will merge all PR-s today, may still request re-review but will merge the current versions of pr

**Pull request merged**